app/cdap/components/Replicator/Detail/index.tsx

```diff
@@ -104,6 +105,7 @@ interface IDetailState {
   setTimeRange: (timeRange: string) => void;
   exportPipeline: () => void;
   duplicatePipeline: () => void;
+  error: object | string | null;
```

Comment:
Not sure about the error object props, but shouldn't we send `error.response` or `error.message` (whichever is in the error object) and remove `object` type here (i.e., restrict it to string | null).

Reply:
error banner handles that

Reply:
Ah I see 👍